app/models/sprint.rb

```diff
@@ -58,17 +58,13 @@ class Sprint < Version
                                                " OR (#{Project.table_name}.lft > #{project.lft} AND #{Project.table_name}.rgt < #{project.rgt} AND #{Version.table_name}.sharing = 'hierarchy')" +
                                                "))"]}}
 
-  scope :displayed_left, lambda { |project| { :joins => sanitize_sql_array(["LEFT OUTER JOIN (SELECT * from #{VersionSetting.table_name}" +
-                                                                                  " WHERE project_id = ? ) version_settings" +
-                                                                                  " ON version_settings.version_id = versions.id",
-                                                                                  project.id]),
-                                                    :conditions => ["(version_settings.project_id = ? AND version_settings.display = ?)" +
-                                                                    " OR (version_settings.project_id is NULL)",
-                                                                    project.id, VersionSetting::DISPLAY_LEFT] } }
+  scope :displayed_left, lambda { |project| {:include => :version_settings,
```

Comment:
Unused block argument - `project`. If it's necessary, use `_` or `_project` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.<br>Use the new Ruby 1.9 hash syntax.<br>Block body expression is on the same line as the block start.<br>Space inside { missing.